nucleus/payara-modules/hazelcast-bootstrap/src/main/java/fish/payara/nucleus/hazelcast/HazelcastCore.java

```diff
@@ -349,10 +368,7 @@ private Config buildConfiguration() {
                 config.setLiteMember(Boolean.parseBoolean(nodeConfig.getLite()));
 
 
-                // set group config
-                GroupConfig gc = config.getGroupConfig();
-                gc.setName(configuration.getClusterGroupName());
-                gc.setPassword(configuration.getClusterGroupPassword());
+                config.setClusterName(configuration.getClusterGroupName());
```

Comment:
Ok, seems this is related, it already did not work ...
https://github.com/hazelcast/hazelcast/issues/11867